DEVELOPER_GUIDE.md

```diff
@@ -24,6 +24,35 @@ Ultimately, your directory structure should look like this:
 │       └── dashboards-assistant
 ```
 
+Setup local stack
+
+1. Export keys locally:
+```sh
+# defaults to us-west-2
+export REGION=
+export AWS_ACCESS_KEY_ID=
+export AWS_SECRET_ACCESS_KEY=
+export AWS_SESSION_TOKEN=
+```
+2. Setup OSD plugins
+```sh
+# from OSD root folder
+cd plugins
+git clone https://github.com/opensearch-project/dashboards-assistant
+(cd dashboards-assistant && yarn setup)
+
+# example with security plugin
+git clone --depth 1 --branch $VERSION https://github.com/opensearch-project/security-dashboards-plugin.git
+
+cd ../../
+yarn osd bootstrap
+```
+2. Start local environment
+```sh
+# from OSD root folder
+yarn start:assistant
```

Comment:
i understand that this makes development easier, but would a plugin specific start command make it less transparent to what's going on? e.g. how does developers tell the difference between `yarn start` and `yarn start:assistant`

maybe add a comment to code reference on what this does?

Reply:
good call. added a comment. this outputs to the terminal each step but it really is different than `start` so the docs could be confusing when reading the difference.